Add cctv command

Squidysquid1 opened the pull request:
Added cctv command as posted by the suggestion in discord.

I did not know what to do about the different locales but I just put them to English to be translated at a later date.

FaiThiX commented:
looks good

alexemanuelol commented on src/commands/cctv.js:
```diff
@@ -0,0 +1,36 @@
+			.setDescription(client.intlGet(guildId, 'commandsCctvDesc'))
+            .addStringOption(option =>
+                option.setName('monument')
+                    .setDescription('Rust Monument')
```
Perhaps add this to en.json translation file as well.

alexemanuelol commented on src/commands/cctv.js:
```diff
@@ -0,0 +1,36 @@
+                    .setDescription('Rust Monument')
+                    .setRequired(true)
+                    .addChoices(
+                        { name: 'Abandoned Military Base', value: 'Abandoned Military Base' },
```
I think there are translations for the different monuments already so use them instead of just using the English one.

alexemanuelol commented on src/discordTools/discordEmbeds.js:
```diff
@@ -846,4 +846,20 @@ module.exports = {
     },
+
+    getCctvEmbed: function (guildId, monument, cctvCodes, dynamic) {
+        let  code = '';
```
two spaces? 😬

alexemanuelol commented on src/discordTools/discordEmbeds.js:
```diff
@@ -846,4 +846,20 @@ module.exports = {
+            code += `${cctvCode} \n`;
+        }
+        if (dynamic) {
+            code += '*\'s means that you need a numarical code that is different for every map';
```
Add this string to translation as well.

alexemanuelol commented on src/discordTools/discordEmbeds.js:
```diff
@@ -846,4 +846,20 @@ module.exports = {
+        return module.exports.getEmbed({
+            color: Constants.COLOR_DEFAULT,
+            timestamp: true,
+            title: `${monument} CCTV Codes`,
```
Perhaps add "codes" to translation?

alexemanuelol commented:
I think you only need to add translation strings to the en.json, once merged, I will sync with crowdin and an automatic PR will be created with the newly added strings.

Could you perhaps add the license header to both .js files?

Squidysquid1 commented:
Added all the recommended translations.  And removed the previous translations from the different languages. Tested command and all is working.

Please let me know if there's anything else that can be done to make it ready to be merged!

alexemanuelol commented on src/structures/DiscordBot.js:
```diff
@@ -28,6 +28,7 @@ const DiscordEmbeds = require('../discordTools/discordEmbeds.js');
 const DiscordTools = require('../discordTools/discordTools');
 const InstanceUtils = require('../util/instanceUtils.js');
 const Items = require('./Items');
+const Cctv = require('./Cctv');
```
Place this in alphabetic order

alexemanuelol commented:
If you'd like, you can add your name and/or github nick to the license underneath my name.

Otherwise all is good 👍
